platform/openide.windows/src/org/openide/windows/WindowManager.java

```diff
@@ -741,6 +741,35 @@ public String getRole() {
         return null;
     }
     
+    /**
+     * An extension to enable client code to manage
+     * Modes directly.
+     * 
+     * @see ModeXml
+     */
```

Comment:
Missing `@since` tag.

Reply:
Hi Jaroslav,

What should I change the version to? Currently manifest contents are:
```
Manifest-Version: 1.0
OpenIDE-Module: org.openide.windows
OpenIDE-Module-Specification-Version: 6.80
OpenIDE-Module-Localizing-Bundle: org/openide/windows/Bundle.properties
AutoUpdate-Essential-Module: true
```

Reply:
Changing it to 6.81  Hope that's correct.

Reply:
Not anymore. The current version is `6.81`. You need to add something - at least `6.82`.

Reply:
OK - changed to 6.82 just now.